src/tests/ftest/util/soak_utils.py

```diff
@@ -70,11 +70,14 @@ def add_containers(self, pool, file_oclass=None, dir_oclass=None, path="/run/con
     """Create a list of containers that the various jobs use for storage.
 
     Args:
-        pool: pool to create container
-        file_oclass: file oclass for daos container cmd
-        dir oclass: directory oclass for daos container cmd
-
+        pool (TestPool obj): pool to read/write random data file
+        file_oclass (str): file oclass for daos container cmd
+        dir oclass (str): directory oclass for daos container cmd
+        path (str): namespace for container
     """
+    unique = get_random_string(5, self.used)
+    label = "-".join(["TestContainer", unique])
+    self.used.append(unique)
```

Comment:
The entire contents of this method can be replaced with:
```suggestion
    kwargs = {}
    if file_oclass:
        kwargs['file_oclass'] = file_oclass
        properties = self.params.get('properties', path, "")
        redundancy_factor = extract_redundancy_factor(file_oclass)
        rd_fac = 'rd_fac:{}'.format(str(redundancy_factor))
        kwargs['properties'] = (",").join(filter(None, [properties, rd_fac]))
    if dir_oclass:
        kwargs['dir_oclass'] = dir_oclass
    self.container.append(self.get_container(pool, path, **kwargs))
```